var/spack/repos/builtin/packages/py-fenics-ffc/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFenicsFfc(PythonPackage):
+    """The FEniCS Form Compiler FFC is a compiler for finite element
+    variational forms, translating high-level mathematical descriptions
+    of variational forms into efficient low-level C++ code for finite
+    element assembly."""
+
+    homepage = "https://fenicsproject.org/"
+    git      = "https://bitbucket.org/fenics-project/ffc.git"
+    url      = "https://bitbucket.org/fenics-project/ffc/downloads/ffc-2019.1.0.post0.tar.gz"
+
+    version('2019.1.0.post0', sha256='306e1179630200a34202975a5369194939b3482eebfc34bc44ad74dab1f109e8')
+    version('2018.1.0',       sha256='c5a6511693106d1cd2fc013148d0cd01cd1b99fc65dab461ca0b95851a9ea271')
+    version('2017.2.0.post0', sha256='1969a5460cb866c478df64874ce213f81cb5c893b89f991a578e258b1a64fee5')
+    version('2016.2.0',       sha256='916c94c18047e1281d08e67c515aa1d4175320723609fbbfb0fc498ab4d41bb6')
+    version('1.6.0',          sha256='418b0b89f2e46e28611476f8269d99461c57a79b8c4e676821c2a063d3ed26c2')
+
+    depends_on('python@3:', type=('build', 'run'))
+
+    depends_on('py-setuptools', type='build')
+
+    depends_on('py-numpy')
```

Comment:
Shouldn't this be type=build/run or build/link/run?

Reply:
Cross checking with the `py-fenics-ffcx` package I agree this should be of type `(build, run)`. I'm not sure if it is required as it currently works on my machine, however I will align it in the next commit.